tigerpath/majors_and_certificates/degrees/BSE_2018.json

```diff
@@ -328,33 +306,30 @@
           "name": "Seven Total Courses",
           "max_counted": 1,
           "min_needed": 7,
-          "description": "7 total courses in humanities and social science",
           "explanation": "Candidates for the B.S.E. degree must take a minimum of seven courses in the humanities and social sciences.",
           "double_counting_allowed": true,
           "req_list": [
             {
               "name": null,
               "max_counted": null,
               "min_needed": null,
-              "description": "Epistemology and Cognition (EC)",
-              "explanation": "The requirement in epistemology and cognition will introduce you to the critical study of the nature, sources and bounds of human knowledge. Courses in this category examine topics ranging from the basic perceptual capabilities that humans share with other animals to the distinctively human capacity for language; from theoretical models of human knowledge to empirical models of an individual's cognitive abilities; from the historical record of collective inquiry in the sciences and elsewhere to informed speculations on the outer limits of what is knowable.",
+              "explanation": null,
               "dist_req": "EC"
             },
             {
               "name": null,
               "max_counted": null,
               "min_needed": null,
-              "description": "Ethical Thought and Moral Values (EM)",
-              "explanation": "The requirement in ethical thought and moral values is designed to engage you in disciplined reflection on human conduct, character and ways of life. Through inquiry into questions of ethics and morality as presented in works from one or more cultural traditions, these courses will help you to discern, understand and appreciate ethical issues and to articulate, assess and defend moral judgments in an informed and thoughtful way.",
+              "explanation": null,
               "dist_req": "EM"
             },
             {
               "name": null,
               "max_counted": null,
               "min_needed": null,
-              "description": "Foreign Language — a language course at the 107/108 level or above",
-              "explanation": "When you become proficient in a foreign language, you acquire more than a communication skill; you become literate in another culture and gain another perspective on the world.",
+              "explanation": null,
               "course_list": [
+                "LANG 1027",
```

Comment:
French and Italian both have a 1027-level double course, which combines 102 and 107 and completes the language requirement, but was previously not counting.
This fixes it.